jetty9-base/src/main/jetty-config/jetty.commands

```diff
@@ -12,3 +12,6 @@ jetty.server.stopTimeout=30000
 jetty.httpConfig.securePort=443
 
 jetty.deploy.scanInterval=0
+
+#TODO remove this.  Needed to fix configuration error in jetty-9.4.0, fixed in 9.4.1
+jetty.httpConfig.forwardedHttpsHeader=X-Forwarded-For
```

Comment:
This is a bug in 9.4.0 config where the value of forwardedHttpsHeader property is set as the X-Forwarded-For.  Fixed in 9.4.1, which will hopefully be out soon, so perhaps wait until then rather than commit this work around.